### DISCUSSION TITLE
possible to use audible from amazon?

### DISCUSSION THREAD

**Discussion by pogojens** [MiczFlor/RPi-Jukebox-RFID#1715] [Q&A]
Hey everyone :)
First things first: Many thanks to this really nice Project! Great work an efford!
I do have an spotify and an audible account. Is it possible to use audible as well? 
Has anybody any experience with that?
best regards,
Jens

**Comment by lroeser** [comment]
Hello, well Mopidy doenst seem to offer a audible extension, so this will not work out-of-the-box.
What you can do however is convert your audible books to mp3 and use these with phoniebox.
There are multiple tools and online-converters for the conversion.
